bundles/admin/javascript-hooks/view.rst

```diff
@@ -0,0 +1,23 @@
+View
+====
+
+The view-hook is just a boolean flag which resets the layout to its default values (if no
+own layout-hook is set).
+
+The defaults of the layout-hook are:
+
+.. code-block:: javascript
+
+    layout: {
+        navigation: {
+            collapsed: false,
+            hidden: false
+        },
+        content: {
+            width: 'fixed',
+            leftSpace: true,
+            rightSpace: true,
+            topSpace: true
+        },
+        sidebar: false
+    }
```

Comment:
Is this a good idea at all? There are many components with this view parameter, could lead to strange behaviour IMO. Wouldn't it better to just set an empty object to layout for resetting it? Using a different parameter for that confuses a bit.